pyomo/gdp/plugins/bigm.py

```diff
@@ -530,8 +530,14 @@ def _transfer_transBlock_data(self, fromBlock, toBlock):
             original._transformation_block = weakref_ref(newblock)
             newblock._srcDisjunct = weakref_ref(original)
 
-        # we delete this container because we just moved everything out
-        del fromBlock.relaxedDisjuncts
+            # save index of what we just moved so that we can delete it
+            to_delete.append(idx)
+
+        # delete everything we moved.
+        for idx in to_delete:
+            # Note we have to do it by index because python scoping is
+            # obnoxious...
```

Comment:
Huh? What do you mean?

Reply:
Eh, nothing, I just forgot to remove my annoyance at how hard it is to delete something in python sometimes...